x-pack/solutions/observability/plugins/infra/public/components/asset_details/tabs/overview/metadata_summary/metadata_summary_list.tsx

```diff
@@ -190,28 +125,31 @@ const MetadataSummaryListWrapper = ({
     </Section>
   );
 };
-export const MetadataSummaryList = ({ metadata, loading, entityType }: MetadataSummaryProps) => {
+export const MetadataSummaryList = ({
+  metadata,
+  loading,
+  entityType,
+  schema,
+}: MetadataSummaryProps) => {
+  const { host, container } = getMetadataBySchema(metadata?.info, schema);
```

Comment:
I'm still not convinced about this, getting container metadata based on schema when we know for sure we won't have that functionality for now. Someone that is not aware of the work we are doing could check the code and expect to get schema for containers too